packages/block-editor/src/components/line-height-control/index.js

```diff
@@ -85,12 +85,17 @@ const LineHeightControl = ( {
 		: { marginBottom: 24 };
 
 	const handleOnChange = ( nextValue, { event } ) => {
+		if ( nextValue === '' ) {
+			onChange();
+			return;
+		}
```

Comment:
This is needed if we land https://github.com/WordPress/gutenberg/pull/49750 which changes `cleanEmptyObject` to keep anything that `!== undefined`. 

It also allows the following onChange calls to simply convert the value to a string improving zero value handling.